logging/logging_test.go

```diff
@@ -209,6 +209,55 @@ func TestLogSync(t *testing.T) {
 	}
 }
 
+func TestLogSyncWithMinimalLoggedSeverity(t *testing.T) {
```

Comment:
nit: consider refactoring instead of creating just another copy of the same code.